fns-hl7-pipeline/fn-receiver-debatcher/pom.xml

```diff
@@ -25,7 +25,7 @@
         <stagingDirectory>${project.build.directory}/azure-functions/${functionAppName}</stagingDirectory>
         <gson.version>2.10.1</gson.version>
         <hl7pet.version>1.2.7.1</hl7pet.version>
-        <lib-dex-commons.version>0.0.39-SNAPSHOT</lib-dex-commons.version>
+        <lib-dex-commons.version>0.0.41-SNAPSHOT</lib-dex-commons.version>
```

Comment:
I thought we were naming the versions according to the Sprint release number? 0.0.41 is a few weeks away.
Is it possible to add a minor version to the end instead? 0.0.39.1 ?

Reply:
libraries don't have to follow the release number - i had to release that twice on a single sprint and if i dn't change the version, you can get caching problems!

Reply:
just noticed your suggestion but no - we are following SEM version and there's no 4th number :)